README.md

```diff
@@ -46,33 +46,29 @@ The CLI currently supports managing `cStor`, `Jiva`, `LocalPV-LVM`, `LocalPV-ZFS
 * [Contributing](#contributing)
 
 
-### Installation
+## Installation
 
 OpenEBSCTL is available on Linux, macOS and Windows platforms.
 
+* (**Recommended**) The latest binary can be installed via `krew`
+```bash
+$ kubectl krew install openebs
+...
+...
+$ kubectl krew list
+PLUGIN    VERSION
+openebs    v0.2.0
+...
+...
+# to update the openebs plugin
+$ kubectl krew upgrade openebs
+...
+...
+```
+
 * Binaries for Linux, Mac and Windows are available as tarballs and zip in the [release](https://github.com/openebs/openebsctl/releases) page.
-* For Linux, download the respective tarball from [release](https://github.com/openebs/openebsctl/releases) page and :-
-   ```shell
-   tar -xvf kubectl-openebs_v0.2.0_Linux_x86_64.tar.gz
-   cd kubectl-openebs_v0.2.0_Linux_x86_64
-   sudo mv kubectl-openebs /usr/local/bin/
-   ```
-  Or, download the `debian` package from the [release](https://github.com/openebs/openebsctl/releases) page and double click it launch the installer if using GUI.<br/><br/>
-  Or, we can also use script to install the latest version :-
-   ```shell
-   wget https://raw.githubusercontent.com/openebs/openebsctl/develop/scripts/install-latest.sh -O - | bash
-   ```
-* For Mac, download the respective tarball from [release](https://github.com/openebs/openebsctl/releases) page and :-
```

Comment:
Do we want to remove the steps of installation for mac and windows?

Reply:
Yes, krew is the best way to download & update the plugin. Besides, once people use krew going to the releases page won't make sense.

Reply:
Okay. Awesome